src/main/scala/scalismo/faces/parameters/RenderObject.scala

```diff
@@ -141,6 +143,20 @@ object MoMoInstance {
            modelURI: URI): MoMoInstance = {
     fromCoefficients(MoMoCoefficients.zeros(shapeComponents, colorComponents, expressionComponents), modelURI)
   }
+
+  /** create a random coefficient vector with its elements N(0,1) distributed. */
+  def sample(model: MoMo, modelURI: URI)(implicit rnd: Random): MoMoInstance = {
+    val fullShapeCoeffs = for (_ <- 0 until model.neutralModel.shape.rank) yield Gaussian(0, 1).draw()
+    val fullColorCoeffs = for (_ <- 0 until model.neutralModel.color.rank) yield Gaussian(0, 1).draw()
+    val expressionRank = model.expressionModel.get.expression.rank
+    val coeff = if(expressionRank > 0) {
```

Comment:
You do not need to check. The if and else branch do exactly the same for values smaller or equal zero.